vpc/proto/vpc.proto

```diff
@@ -245,6 +245,21 @@ message GCLegacySetupResponse {
     repeated NetworkInterfaceAttachment networkInterfaceAttachment = 1;
 }
 
+message LockId {
+    int64 id = 1;
+}
+
+message Lock {
+    int64 id = 1;
+    string lockName = 2;
```

Comment:
This makes me thing that longer-term we should probably put the keyedItem + lock worker name into the database just as useful metadata.